doc/source/io.rst

```diff
@@ -4018,6 +4019,43 @@ easy conversion to and from pandas.
 
 .. _xray: http://xray.readthedocs.org/
 
+.. _io.sas:
+
+SAS Format
+----------
+
+.. versionadded:: 0.17.0
+
+The top-level function ``read_sas`` currently can read (but not write)
+SAS xport (.XPT) format files.  Pandas cannot currently handle
+SAS7BDAT files.  XPORT files only contain two value types: ASCII text
+and double precision numeric values.  There is no automatic type
+conversion to integers, dates, or categoricals.  By default the whole
+file is read and returned as a ``DataFrame``.  Specify a ``chunksize``
```

Comment:
add the ``chunksize/itertor`` in a small section (e.g. you can use ``~~~~~`` to break this up a bit)